src/cmd/singularity/cli/verify.go

```diff
@@ -40,3 +41,18 @@ var VerifyCmd = &cobra.Command{
 	Long:    docs.VerifyLong,
 	Example: docs.VerifyExample,
 }
+
+func doVerifyCmd(cpath, url string) error {
+	if url == "" {
+		// lookup key management server URL from singularity.conf
+
+		// else use default builtin
+		url = defaultKeysServer
```

Comment:
Same user confusion comment here. Suggest putting this logic in `init()`.

Reply:
This is removed completely see my comment above.